docs/noise/qcvv/parallel_xeb.ipynb

```diff
@@ -91,6 +91,44 @@
     "import numpy as np"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***Note: The entire XEB workflow can be run by calling `cirq.experiments.parallel_two_qubit_xeb` and the combined single-qubit randomized benchmarking (RB) and XEB workflows can be run by calling `cirq.experiments.run_rb_and_xeb`.***"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qubits = cirq.GridQubit.rect(3, 2, 4, 3)\n",
+    "result = cirq.experiments.parallel_two_qubit_xeb(\n",
+    "    sampler=cirq.DensityMatrixSimulator(noise=cirq.depolarize(5e-3)),  # Any simulator or a ProcessorSampler.\n",
+    "    qubits=qubits \n",
+    ")\n",
+    "\n",
+    "# The returned result is an instance of the `TwoQubitXEBResult` class which provides visualization methods like \n",
```

Comment:
Please split this to 3 cells for (i) simulation, (ii) plotting, (iii) results printout.
Users will be then able to play with visualizations without having to wait for computation.

Reply:
done